spec/openapi.yml

```diff
@@ -1069,8 +1112,16 @@ components:
         args:
           description: The arguments of the run.
           type: object
+        facets:
+          $ref: '#/components/schemas/runFacets'
       example:
         args: {'email': 'me@example.com', 'emailOnFailure': false, 'emailOnRetry': true, 'retries': 1}
+        facets: {
+          'context': {
```

Comment:
I would favor deprecating the job `context` field in a future release of Marquez. The context field had a similar goal as facets: _to allow for extra metadata to be sent along with job metadata not part of the core model_. I've opened an issue to start communicating the transition: https://github.com/MarquezProject/marquez/issues/910

Reply:
As this change is deprecating the context field for the future release of Marquez, it sounds like this is aligned?
By defining a `context` facet, it has to be nested under `facets`.

Reply:
Why not just deprecate the `context`? We can communicate the release when the context support will be dropped. Having both `context` and `facets` to extend the metadata for the core models is confusing.

Reply:
I see, I miss understood. We'll be defining the `context` as a _facet_ and deprecating the usage of `Job.context`. This makes sense 👍. I think some keys within the context can become a facet, for example the _sql_ for the job.